.github/workflows/integrationTests.yaml

```diff
@@ -50,11 +50,11 @@ jobs:
         id: date
         run: echo "::set-output name=date::$(date +'%Y-%m-%d' --utc)"
       - uses: actions/checkout@v2
-      - name: Setup Java 11
+      - name: Setup Java 17
         uses: actions/setup-java@v1
         if: matrix.it != 'native'
         with:
-          java-version: 11
+          java-version: 17
       - name: Setup GraalVM
```

Comment:
[Native Support]: Since we're planning on removing native configurations from the repo, the GraalVM job code may also need to be removed from our workflow to complete the cleanup.

Reply:
Thanks for point it out, I'll modify the file accordingly.

Reply:
done.